### DISCUSSION TITLE
EPG on Node (unzipped not installed)

### DISCUSSION THREAD

**Discussion by hackwin** [iptv-org/community#1033] [Q&A]
Can you post the instructions for installing the EPG script for Node.js that is not installed, just unzipped to a folder?

**Comment by freearhey** [comment]
The easiest way to install Node.js is to download the installer for your operating system from their official website: https://nodejs.org/en/download

![image](https://github.com/user-attachments/assets/84758450-9ee6-4e6f-848a-2d7a1c9483c3)

Then run it and follow the instructions on the screen.

The rest of the process is described here: https://github.com/iptv-org/epg?tab=readme-ov-file#installation
